### DISCUSSION TITLE
Hardware volume but software mute control

### DISCUSSION THREAD

**Discussion by mczerski** [snapcast/snapcast#1018] [Q&A]
Hi all,

I have multiple network players around the hous built on top of nano pis. I run on them multiple audio sources like mopidy, spotify connect, bluetooth and snapcast. I want to use single source of volume control so basically all of the sources should control actual alsa mixer volume. And this all works ! But with snapcast is one exception, i dont want it to mute the actual alsa mixer volume, instead i want to be able to disable snapcast as audio source for particular player (to be able to play othersources). And that actually also works but only for players tha DO NOT HAVE MUTE CAPABILITY on their soundcards, and one of my devices unfortunetely has... so is itpossible some how to configure snapcast to use hardvare volume control and software mute control ?
One other thing is that i use mopidy-iris web frontent for mopidy which has support for snapcast and from it i can controll all other devices volume and mute which is veeeeery handy.

**Comment by hdh75** [comment]
hi @mczerski !

Can you share how you disable snapcast as audio source when sending the mute command? That is exactly what I am looking for as I have a device that only can be controlled by softvol which does not support muting.

And that could also be a potential solution for you. If you create a softvol alsa control, you can access it through alsamixer as if it was hardware

Thanks in advance!
